tests/purs/failing/FoldableInstance9.out

```diff
@@ -0,0 +1,50 @@
+Error found:
+in module [33mFoldableInstance9[0m
+at tests/purs/failing/FoldableInstance9.purs:53:1 - 53:38 (line 53, column 1 - line 53, column 38)
+
+  The following data constructors and their subterms must use the type variable 'a' only as the last argument of a data type:
```

Comment:
This list is still incomplete; all the occurrences of `a` in `Test3`, `Test4`, `Test6`, and `Test7` are issues except for the final one in each constructor.